.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,6 @@ Describe the issue you have been trying to solve.
 
 Mention briefly how you have applied the fix.
 
-**Fixes issue:  #[]**
+Fixes #123
```

Comment:
I recommend changing this to `Fixes #`. Otherwise it would be quite easy to miss that the issue they said they are fixing is actually 123. :-)

Reply:
@carolynvs thanks - I like that! fixed in https://github.com/gomods/athens/pull/743/commits/1b2c9a1da6a726b75073c0e850cd928708c66bf2